inference-engine/ie_bridges/c/samples/hello_classification_batch/README.md

```diff
@@ -0,0 +1,28 @@
+# Hello Classification Batch C Sample {#openvino_inference_engine_ie_bridges_c_samples_hello_classification_batch_README}
+
+This topic describes how to run the Hello Classification Batch C sample application.
```

Comment:
do we need a new sample? can we add additional functionality to existing one?

Reply:
You are right. Now, as #3875 was merged, I am closing this PR. Thanks!